File: 프로그래머스/0/120806. 두 수의 나눗셈/두 수의 나눗셈.py
def solution(num1, num2):
    # 출력 결과가 정수를 요구하므로 int()함수를 사용해야 한다.
    return int(num1 / num2 * 1000)

print(solution(3, 2))      # 결과값 : 1500
print(solution(7, 3))      # 결과값 : 2333

File: 프로그래머스/0/120806. 두 수의 나눗셈/두 수의 나눗셈.java
public class Solution {
    public int solution(int num1, int num2) {
        return (int)((double) num1 / num2 * 1000);
    }

    public static void main(String[] args) {
        Solution s = new Solution();
        System.out.println(s.solution(10, 5));
        System.out.println(s.solution(7, 2));
    }
}